development/server/monitor-server-python/monitor-api-python/wsgi.py

```diff
@@ -0,0 +1,4 @@
+from monitor import app
+
+if __name__ == "__main__":
```

Comment:
it would be good to maintain a debug version without gunicorn?

Reply:
Yes, done in last commit.